src/Grid.js

```diff
@@ -0,0 +1,184 @@
+/** @jsx jsx */
+import React from 'react';
+import PropTypes from 'prop-types';
+// import { css, cx } from 'react-emotion';
+import { css, jsx } from '@emotion/core';
+import config from './GridConfig';
+
+const {
+  gridMaxWidth,
+  columnNumber,
+  gutterWidth,
+  mobileBreakpoint,
+} = config;
+
+const columnWidthPX = gridMaxWidth / columnNumber;
+const columnWidth = (columnWidthPX / gridMaxWidth) * 100;
+
+const GridContainerStyles = css`
```

Comment:
Since these `styles` are simple objects/strings, could we change them from `PascalCase` to `camelCase`?